pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/OffloadPoliciesImpl.java

```diff
@@ -248,8 +249,7 @@ public static OffloadPoliciesImpl create(String driver, String region, String bu
 
     public static OffloadPoliciesImpl create(Properties properties) {
         OffloadPoliciesImpl data = new OffloadPoliciesImpl();
-        Field[] fields = OffloadPoliciesImpl.class.getDeclaredFields();
-        Arrays.stream(fields).forEach(f -> {
+        for (Field f : CONFIGURATION_FIELDS) {
```

Comment:
Since we no longer use `stream`, the code `try {} catch{}` can be moved out of the loop, right?

Reply:
The main thing here is to convert the exception to RuntimeException.